installation/src/Controller/InstallationController.php

```diff
@@ -299,4 +299,44 @@ public function delete()
         // We can't send a response with sendJsonResponse because our installation classes now do not exist
         echo json_encode(['error' => false]);
     }
+
+    /**
+     * Opt out from automated updartes
+     *
+     * @return  void
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    public function disableAutomatedUpdates()
+    {
+        $this->checkValidToken();
+
+        /** @var \Joomla\CMS\Installation\Model\AutomatedUpdatesModel $model */
+        $model = $this->getModel('AutomatedUpdates');
+
+        if (!$model->disable()) {
+            // We can't send a response with sendJsonResponse because our installation classes might not now exist
+            $error = [
+                'token' => Session::getFormToken(true),
+                'error' => true,
+                'data'  => [
+                    'view' => 'remove',
+                ],
+                'messages' => [
+                    'warning' => [
+                        Text::sprintf('INSTL_COMPLETE_ERROR_AUTOMATED_UPDATES_DISABLE'),
+                    ],
+                ],
+            ];
+
+            echo json_encode($error);
+
+            return;
+        }
+
+        $this->app->getSession()->destroy();
+
+        // We can't send a response with sendJsonResponse because our installation classes now do not exist
```

Comment:
```suggestion
        // We can't send a response with sendJsonResponse because our installation classes do not exist yet
```

assuming i understand correctly